src/adapter/4C_adapter_porofluid_pressure_based_wrapper.hpp

```diff
@@ -24,7 +24,7 @@ namespace Core::FE
 
 namespace PoroPressureBased
 {
-  class TimIntImpl;
+  class Porofluid;
 }
```

Comment:
This seems to be the central class for the solution procedure so why not `TimeIntegrator`, `Operator`, `Algorithm` etc. I don't really understand the name `PoroPressureBased::Porofluid`. The class is too large to name it so your name may be as good or bad as my suggestions but I stumbled over this.